elide-model-config/src/main/java/com/yahoo/elide/modelconfig/model/TableSource.java

```diff
@@ -35,7 +36,8 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
-public class TableSource {
```

Comment:
Why would these classes need serialization?  They are just HJSon Configuration?

Reply:
It would be nice if RedisCache didn't require changes to the underlying model classes to work.  Maybe there is an alternative serialization scheme where we don't need to modify the source classes.